system/settings/settings.xml

```diff
@@ -564,6 +564,14 @@
           </constraints>
           <control type="list" format="string" />
         </setting>
+        <setting id="videoplayer.adjustresolution" type="boolean" label="13549" help="36163">
```

Comment:
can you think of any reason why you don't want to switch to 2160 if you have a tv that supports it and refresh rate matches?